src/stylesheets/global.scss

```diff
@@ -24,3 +24,7 @@
     @include basic-focus;
   }
 }
+
+html {
+  scroll-behavior: smooth;
```

Comment:
Why do you need this?

Reply:
It was for the feature's original behavior, where the list would automatically scroll down to the search result; don't need it anymore.